### PR TITLE
RNA-Seq analysis with AskOmics interactive tool

### PR DESCRIPTION
**Pull request by xgaia**
Hello everyone,

This is the tutorial to use the [AskOmics IT](https://github.com/galaxyproject/galaxy/pull/9915) in Galaxy.

The tutorial use data from the [RNA-seq counts to genes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html) tutorial.

The AskOmics IT is [not available yet](https://github.com/galaxyproject/galaxy/pull/9915) in Galaxy, but it is available on [galaxy.genouest.org](https://galaxy.genouest.org).

Have a nice day,

Xavier

(ping @abretaud and @bgruening)

### CODE REVIEW

File: topics/transcriptomics/tutorials/rna-seq-analysis-with-askomics-it/tutorial.bib [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#
+# To add bibliography (bibtex) entries here, follow these steps:
+#  1) Find the DOI for the article you want to cite
+#  2) Go to https://doi2bib.org and fill in the DOI
+#  3) Copy the resulting bibtex entry into this file
+#
+# To cite the example below, in your tutorial.md file
+# use {% Batut2018 %}
+
+@article{Batut2018,
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/rna-seq-analysis-with-askomics-it/tutorial.bib:12]: Please remove the unused citation

Comment (xgaia) [reply topics/transcriptomics/tutorials/rna-seq-analysis-with-askomics-it/tutorial.bib]: Done

**Comment by xgaia** [comment]
> You might want to add pre-requirements? Maybe the RNA seq tutorial and add to the RNA seq tuotorial an annotation to look at this tutorial next. This will enable a learning path. See here https://github.com/galaxyproject/training-material/edit/master/topics/transcriptomics/tutorials/ref-based/tutorial.md follow_up_training ... maybe?

I added `rna-seq-counts-to-genes` as a requirement in mh tutorial

ANd I added my tutorial as a follow up training to `rna-seq-counts-to-genes` (plus visualizations tutorials)

**Comment by bgruening** [comment]
It should be deployed on EU by now.

**Comment by bgruening** [comment]
Thanks @xgaia thanks a lot!!!!

**Pull request merged**